File: control-flow/Q135/main.go
package main

import "fmt"

func main() {
	for i := 2; i <= 14; i += 4 {
		temp := i
		for j := 1; j <= 6; j++ {
			fmt.Printf("%d\t", temp)
			temp += 2
		}
		fmt.Println()
	}
}

File: control-flow/Q135/main.rs
use std::io;

fn main() {
    let _ = io::stdin();
    for i in (2..=14).step_by(4) {
        let mut temp = i;
        for _ in 1..=6 {
            print!("{}\t", temp);
            temp += 2;
        }
        println!();
    }
}